test_googleMapsApiModule.py

```diff
@@ -0,0 +1,26 @@
+import googleMapsApiModule
```

Comment:
Wouldn't it make sense to have this class in the same file? Thoughts @janakrajchadha @athityakumar

Reply:
@vishakha-lall should I include some comments on the details of tests, or should I describe them here ?

Reply:
@vishakha-lall - Usually we have `src` and `test` directories to separate out the code and testing modules. We could adopt it here too 😄

Reply:
However, I'm thinking if this refactoring should be within this PR or as a different PR. Because it'd create potential conflicts with other PRs as well

Reply:
@athityakumar as discussed in the channel, we can do the refactoring in another PR.Thanks.